test/integration/enumerating-databases/list_databases.test.ts

```diff
@@ -0,0 +1,77 @@
+import { expect } from 'chai';
+
+import { AddUserOptions, MongoClient, MongoServerError } from '../../../src';
+
+describe('listDatabases', function () {
+  beforeEach(function () {
+    if (process.env.AUTH !== 'auth') {
+      this.skip();
+    }
+  });
+
+  describe('authorizedDatabases flag', function () {
+    const username = 'a';
+    const password = 'b';
+    const mockAuthorizedDb = 'mockAuthorizedDb';
+
+    let client, authorizedClient: MongoClient;
+
+    const authorizedUserOptions: AddUserOptions = {
+      roles: [{ role: 'read', db: mockAuthorizedDb }]
+    };
+
+    beforeEach(async function () {
+      client = this.configuration.newClient({ auth: { username: 'user', password: 'password' } });
```

Comment:
Can we add a coment that these username and pass comes form mlaunch's defaults. 
When we fix the CI issue we'll prob have to change these to fetched from the environment